include/zephyr/net/wifi_mgmt.h

```diff
@@ -588,6 +595,16 @@ struct wifi_reg_chan_info {
 	unsigned short dfs:1;
 } __packed;
 
+/** Wi-Fi AP parameters */
+struct wifi_ap_params {
+	/** Interface index */
+	uint8_t if_index;
+	/** Frequency band */
+	enum wifi_frequency_bandwidths bandwidth;
+	/** Get or set operation */
+	enum wifi_mgmt_op oper;
```

Comment:
Please change the ordering so that the if_index is the last option. This makes no real difference now, but if we add new parameters in this struct we could avoid holes in the memory layout.

Reply:
Okay